paper/paper.md

```diff
@@ -43,24 +43,64 @@ When working with Bayesian models there are a series of related tasks that need
+from PPLs as an xarray [@HoyerxarrayNDlabeled2017] dataset, ArviZ provides labeled querying of the
+data, efficient algorithms, and persistent metadata. These datasets are stored together on disk and
+in code using netCDF4 [@56302;@brown_1993] groups, which are themselves built with HDF5, and allows
+for well supported serialization. This functionality is implemented in the InferenceData class (see Figure 1).
+In addition to the listed benefits of using netCDF and xarray, by using a single data structure all
+statistical and visualization functions need to be implemented only once.
 
-![groups](https://d2mxuefqeaa7sj.cloudfront.net/s_26E7E0D1516EA1B427269A258102C3AC9090025345CBB4CA6C7DBDA445D6595F_1542830805296_inference_data.png)
+![Relationship between netCDF, az.InferenceData, and xarray](https://d2mxuefqeaa7sj.cloudfront.net/s_26E7E0D1516EA1B427269A258102C3AC9090025345CBB4CA6C7DBDA445D6595F_1542830805296_inference_data.png)
 
 
-In addition to common plots for Bayesian analysis including a trace plot and forest plot, ArviZ implements other visualizations such as a plot for posterior predictive checks, a pair plot, and a parallel coordinate plot [@GabryVisualizationBayesianworkflow2017]. Additionally, it supports a number of statistical checks, such as calculating the effective sample size, the r-hat statistic, Pareto-smoothed importance sampling leave-one-out cross validation (PSIS-LOO-CV) [@VehtariPracticalBayesianModel2015], and widely applicable information criterion (WAIC) [@watanabe_widely_2013].
+In addition to common plots for Bayesian analysis including a trace plot and forest plot,
+ArviZ implements other visualizations such as a plot for posterior predictive checks, a pair plot,
+and a parallel coordinate plot [@GabryVisualizationBayesianworkflow2017]. Additionally, it supports
+a number of statistical checks, such as calculating the effective sample size, the r-hat statistic,
+Pareto-smoothed importance sampling leave-one-out cross validation (PSIS-LOO-CV)
+[@VehtariPracticalBayesianModel2015], and widely applicable information criterion (WAIC)
+[@watanabe_widely_2013].  
+
+# Example plots  
+Some sample plots are shown in Figure 2 through Figure 5.  
+![Bivariate hexbin plot with marginal distributions](plot_joint.png)  
+
+![2D Kernel Density estimation](plot_kde_2d.png)  
+
+![Markov Chain Monte Carlo Trace Plot](plot_trace.png)  
+
+![John Kruschke styled posterior distribution plots](plot_posterior.png)  
+
 
 # Funding
 
 Work by Osvaldo Martin was supported by CONICET-Argentina and ANPCyT-Argentina (PICT-0218).
 
 # Acknowledgments
 
-We thank the PyMC3 Community — especially Adrian Seyboldt, Junpeng Lao, and Thomas Wiecki — as well as the Stan community — especially Allen Riddell . We also would like to extend thanks to the open source ArviZ contributors, and the contributors to the libraries used to build ArviZ — particularly xarray, matplotlib, pandas, and numpy.
```

Comment:
What about removing the "open source" and just write "...thanks to the ArviZ contributors..."?

Reply:
I bet this would be fine, would match verbiage of the sentence below